scripts/mkcloud

```diff
@@ -407,18 +407,24 @@ function setupadmin
     onadmin write_cloud_info
     echo "you can now proceed with installing crowbar"
     # prevent jumbo frames from going out
+    if [[ $(iptables --version) == "iptables v1.4.16.3" ]]; then
```

Comment:
You could alternatively check whether `iptables` accepts the `--wait` option:

~~~
local IPTABLES_WAITOPT=""
if $($sudo iptables --wait --list INPUT > /dev/null); then
  IPTABLES_WAITOPT="-w "
fi
~~~